webpack.config.esm.ts

```diff
@@ -52,7 +52,7 @@ const hasGit = fs.existsSync(path.resolve(__dirname, '.git'));
 // Hack alert: due to a variety of issues, sometimes we need to change
 // the name here. Mostly it's things like webpack changes that affect
 // how minification is done, even though that's supposed not to matter.
-const webpackJsHack = '.v56.';
+const webpackJsHack = '.v57.';
```

Comment:
This might need to be bumped again post-merge as I just bumped it for some other reason